training_day1/src/components/Button.js

```diff
@@ -0,0 +1,18 @@
+import { useState } from "react";
+import 'bootstrap/dist/css/bootstrap.min.css';
+function ButtonComponent() {
+
+    const [name, setName] = useState("sourabh");
+
+    return (
+        <div className="container">
+            <label for="name">Name:  </label>
+            
+            <input className="m-2" type="text" id="name" name="name" value={name}></input>
+            <button className="btn btn-primary" onClick={() => { setName("React") }} >Click Me</button>
```

Comment:
we have to set the name to the value we added in the input tag, correct this logic